q2_feature_table/tests/filter/test_filter_samples.py

```diff
@@ -626,12 +693,12 @@ def test_combine_exclude_ids_and_frequency_filters(self):
         table = Table(np.array([[0, 1, 3], [1, 1, 2]]),
                       ['O1', 'O2'],
                       ['S1', 'S2', 'S3'])
-        actual = filter_samples(table,
-                                metadata=metadata,
-                                exclude_ids=True,
-                                max_frequency=4,
-                                min_frequency=3)
-        self.assertTrue(actual.is_empty())
+        with self.assertRaisesRegex(ValueError, 'table is empty'):
```

Comment:
Same as above comments for comment on line 687/688 and you'll also want to include the True case for this test as well.